packages/round-manager/src/features/round/ViewRoundSettings.tsx

```diff
@@ -888,37 +930,37 @@ function DetailsPage(props: {
-                            className="ml-4"
-                            onClick={() => {
-                              const updatedRequirements = [
-                                ...(props.editedRound!.roundMetadata
-                                  .eligibility!.requirements || []),
-                              ];
-                              updatedRequirements.splice(i, 1);
-                              props.control.unregister(
-                                `roundMetadata.eligibility.requirements.${i}.requirement`
-                              );
-                              props.setEditedRound({
-                                ...props.editedRound,
-                                roundMetadata: {
-                                  ...props.editedRound?.roundMetadata,
-                                  eligibility: {
-                                    ...props.editedRound?.roundMetadata
-                                      .eligibility,
-                                    requirements: updatedRequirements,
-                                    description:
-                                      props.editedRound?.roundMetadata
-                                        .eligibility?.description || "",
-                                  },
+                        <button
+                          data-testid="remove-requirement-button"
+                          className="ml-4"
+                          onClick={() => {
+                            const updatedRequirements = [
+                              ...(props.editedRound!.roundMetadata.eligibility!
+                                .requirements || []),
+                            ];
+                            updatedRequirements.splice(i, 1);
+                            props.control.unregister(
+                              `roundMetadata.eligibility.requirements.${i}.requirement`
+                            );
+                            props.setEditedRound({
+                              ...props.editedRound,
+                              roundMetadata: {
+                                ...props.editedRound?.roundMetadata,
+                                eligibility: {
+                                  ...props.editedRound?.roundMetadata
+                                    .eligibility,
+                                  requirements: updatedRequirements,
+                                  description:
+                                    props.editedRound?.roundMetadata.eligibility
+                                      ?.description || "",
                                 },
-                              });
-                            }}
-                          >
-                            <Cross color="red" />
-                          </button>
-                        )}
+                              },
+                            });
+                          }}
```

Comment:
If the logic in the top of ViewRoundSettings is moved to a hook, this logic could be added to a function.